lib/jobs/base-job.js

```diff
@@ -666,6 +662,67 @@ function baseJobFactory(
             self.subscriptions.push(subscription);
         });
     };
-    
+
+    /**
+     * @function _subscribeUcsCommand
+     * @description subscribe and wait for a request from the ucs command exchange.
+     **/
+    BaseJob.prototype._subscribeUcsCommand = function(uuid, callback) {
```

Comment:
If paired with _publishRunUcsCommand, suggest to change this function name to _subscribe**Run**UcsCommand

Reply:
Fixed, use universal naming for subscribe and publich method on base-job.js.